src/main/java/frc/robot/constants/fields/HomeField.java

```diff
@@ -1,13 +1,18 @@
 package frc.robot.constants.fields;
 
+import edu.wpi.first.wpilibj.geometry.Translation2d;
 import frc.robot.constants.FieldConstants;
 
 /**
- * Home field dimensions and distances.
+ * Home field way-points. relative to the (0,0) point in PathWeaver. all
+ * measurement are measured in meters.
  */
 public class HomeField extends FieldConstants {
+    // TODO: check reality
     public HomeField() {
-        feederConstants.ROCKET_TO_FEEDER = 250;
-        feederConstants.SIDE_WALL_TO_MIDDLE_FEEDER = 50;
+        TRENCH_RUN_ENTRANCE = new Translation2d(5.45, -0.70485);
+        MIDDLE_OF_TRENCH = new Translation2d(9.294, -0.7112);
```

Comment:
It's not entirely clear where exactly is MIDDLE_OF_TRENCH...

Reply:
Trench = control panel thingy